Feature/restructure intro philipp

pdiercks opened the pull request:
Attempting to merge user stories into feature/restructure-introduction branch. This is work in progress and needs revision.

I also made small changes to requirements.rst, mainly because some things are not fully clear and can be confusing since some requirements have multiple aspects (see the comments in the file).

I also added a dodo.py to build the website.

dglaeser commented:
@pdiercks I rebased to remove conflicts and added a commit that makes a few changes on the user story of the reproducible paper guy. I can remove that commit again, no problem. Maybe you want to double-check.

One general question, I don't yet get the function of the names (Sylvester, etc.). Will we refer to them later at one point? They way it is now I think readers get a bit confused maybe?

dglaeser commented on docs/userstories.rst:
```diff
@@ -16,14 +16,16 @@ Names:
 * Jean Reno (Leon der Profi, Mathilda)
 
-In this user story, the output of the workflow is a scientific paper describing the research results.
-An essential component of this work is the development of research software in line with the task area `BETTY <https://nfdi4ing.de/archetypes/betty/>`_.
```
I removed the reference to `Betty` here because I thought this would rather be confusing for readers that haven't yet read about the structure of NFDI4Ing.

dglaeser commented on docs/userstories.rst:
```diff
@@ -16,14 +16,16 @@ Names:
+involved in the research workflow (numerical analysis, postprocessing, etc.).
+
+In order to meet this goal, i.e. the *reproducibilty* and *transparency* of computational research, the following requirements are defined.
+First of all, the code has to published. This can happen in the form of a tarball (e.g. via `Zenodo <https://zenodo.org>`_), or by emplying
```
Here, I differentiated a bit more, because one doesn't necessarily have to employ version control to make the code accessible.

pdiercks commented:
> @pdiercks I rebased to remove conflicts and added a commit that makes a few changes on the user story of the reproducible paper guy. I can remove that commit again, no problem. Maybe you want to double-check.
> 
> One general question, I don't yet get the function of the names (Sylvester, etc.). Will we refer to them later at one point? They way it is now I think readers get a bit confused maybe?

Thanks for removing the conflicts. Currently these names are only suggestions. In the last meeting we discussed if it would be helpful to give these user stories names similar to how the arche types are given names. I would also rather delete the names if we don't find a good one.
